src/htu21d/htu21d.cpp

```diff
@@ -154,12 +154,16 @@ HTU21D::getDewPoint(int bSampleData)
         sampleData();
     }
 
-	float fA  = 8.1332;
-	float fB  = 1762.39;
-	float fC  = 235.66;
-	float fT  = getTemperature(false);
-	float fRH = getHumidity(false);
+    float fA  = 8.1332;
+    float fB  = 1762.39;
+    float fC  = 235.66;
+    float fT  = getTemperature(false);
+    float fRH = getHumidity(false);
+#ifdef __ANDROID__
+    float fPP = pow(10, fA - (fB / (fT + fC)));
+#else
     float fPP = exp10(fA - (fB / (fT + fC)));
+#endif
     float fDP = -(fB / (log10(fRH * fPP / 100) - fA) + fC);
```

Comment:
I wonder if we should just stick to pow() (or perhaps, more appropriately, powf()) here and get rid of exp10()?  exp10 seems to be a gnu extension only.  Then the ifdef would not be necessary?